app/core/crossref/generateCrossRefXml.ts

```diff
@@ -0,0 +1,10 @@
+import { toXml } from "xast-util-to-xml"
+import generateCrossRefObject from "./generateCrossRefObject"
+
+type CrossRefArgs = Parameters<typeof generateCrossRefObject>[0]
```

Comment:
Ah because `Parameters` returns a tuple (like a stricter array) of all the parameters of the function, but the function only has one parameter, so we just need the first item (aka the one parameter)!